src/StaticFAQ/breadcrumbs/__test__/CustomBreadcrumbs.test.js

```diff
@@ -2,10 +2,10 @@
 
 import * as React from 'react';
 import { mount } from 'enzyme';
-import ThemeProvider from '@kiwicom/orbit-components/lib/Theming/ThemeProvider';
 
 import WithRouter from '../../../common/__tests__/helpers/ComponentWithRouter.ignore';
 import CustomBreadcrumbs from '../CustomBreadcrumbs';
+import ThemeProvider from '../../../common/ThemeProvider';
```

Comment:
you don't need `ThemeProvider` in tests anymore anywhere

Reply:
fixed, thanks!